languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -63936,6 +63936,63 @@ USA
                 <example>I <marker>haven't said no</marker> yet.</example>
             </rule>
         </rulegroup>
+        <rulegroup id="DOUBLE_NEG_RP" name="I didn't mention it to no one" default="temp_off">
+            <antipattern>
+                <token regexp="yes">could|would|should</token>
+                <token regexp="yes">not|n't</token>
+                <token postag="VB" />
+            </antipattern>
+            <rule>
+                <antipattern>
+                    <token regexp="yes">come|present</token>
+                    <token regexp="yes">from|in</token>
+                    <token>nothing</token>
+                </antipattern>
+                <pattern>
+                    <token regexp="yes">not|n't</token>
+                    <token postag="VB">
+                        <exception regexp="yes" inflected="yes">die|get|try</exception>
+                    </token>
+                    <token postag="IN|RP|TO" postag_regexp="yes" />
+                    <marker>
+                        <token>nothing</token>
+                    </marker>
+                </pattern>
+                <message>This phrase contains a double negative, or a comma may be missing. Try: <suggestion>anything</suggestion>.</message>
+                <example correction="anything">I won't stop for <marker>nothing</marker>.</example>
+                <example>From nothing there will always be something because a rule that “something can't come from nothing” would be something, and therefore it is not present in nothing.</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <token regexp="yes">not|n't</token>
+                    <token postag="VB" />
+                    <token min="0" chunk_re="B\-NP\-(singular|plural)" />
+                    <token min="0" chunk_re="E\-NP\-(singular|plural)" />
+                    <token postag="IN|RP|TO" postag_regexp="yes" />
+                    <marker>
+                        <token>nobody</token>
+                    </marker>
+                </pattern>
+                <message>This phrase contains a double negative, or a comma may be missing. Try: <suggestion>anybody</suggestion>.</message>
```

Comment:
In general, it's best if you can avoid putting the suggestion (or the suggested form) in the message, so that the suggestion doesn't appear twice (once in the message, and once as the suggestion itself).